### DISCUSSION TITLE
Sharing key mapping files

### DISCUSSION THREAD

**Discussion by guanzhangrtk** [barry-ran/QtScrcpy#417] [Q&A]
Hi, I understand that coordinates for key mapping are relative, however it seems that if the aspect ratios of the devices are different, the coordinates may still be off?

I created key map for a game and when I shared it with someone else it didn't seem to work. On my device the coordinates lands on a button but on the other device it's empty space.

Is there any workaround for this issue? Thanks!

**Comment by dijo7** [comment]
Both the user's phone and windows resolutuion, dpi scaling(in mobile and pc),  and casting resolution  must be same

**Reply by guanzhangrtk** [reply]
I understand that but I wonder if it will be possible to do some conversion such that those models with at least the same aspect ratios will be able to share the same key maps. I believe projects like Airtest IDE does that with relative coordinates positioning.
